pkg/apis/core/validation/shoot_test.go

```diff
@@ -487,6 +487,95 @@ var _ = Describe("Shoot Validation Tests", func() {
 			})
 		})
 
+		Context("#ValidateForceDeletion", func() {
+			It("should not allow setting the force-deletion annotation if the Shoot does not have a deletionTimestamp", func() {
+				newShoot := prepareShootForUpdate(shoot)
+
+				metav1.SetMetaDataAnnotation(&newShoot.ObjectMeta, v1beta1constants.AnnotationConfirmationForceDeletion, "1")
+
+				err := ValidateForceDeletion(newShoot, shoot)
+				Expect(err).To(PointTo(MatchFields(IgnoreExtras, Fields{
```

Comment:
Nit: You could inline this

```suggestion
				Expect(ValidateForceDeletion(newShoot, shoot)).To(PointTo(MatchFields(IgnoreExtras, Fields{
```